### PR TITLE
:books: Added Insomni'hack CTF 2016

### PR DESCRIPTION
**Pull request by bcyrill**
- Created base layout

### CODE REVIEW

**Comment by abpolym** [comment]
Thanks @bcyrill !

File: insomnihack-ctf-2016/.gitignore [hunk 1]
```diff
@@ -0,0 +1,5 @@
+backdoor/smartdoor1/smartdoor1_2ed98791bd003533dc8b4b5ea79a6656
+backdoor/smartdoor2/smartdoor2_2ceb2e595d5f58fa55297e2e1bbc274b
+backdoor/smartdoor3/smartdoor3_6c931f4d5090f6a360bee7db3681c0d3
+hardware/iotcode/iotcode_8dfe573cdbccd541375408706d2c583c.zip
+shellcode/superpollute/qemu-sh4_74ca7af34c287f9d3f20d126bad26fe7.tgz
```
Comment (dhanvi) [inline insomnihack-ctf-2016/.gitignore]: @bcyrill so these file are greater than 10 MB so did you upload them to somewhere else (cloud) and added the link here ?

File: insomnihack-ctf-2016/crypto/pcapbleeding/README.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+**Author:** julien 
+**Difficulty:** Medium
+**Description:**
+
+> Finally, somebody has read our 2014 log files. We think an attack occurred on our HTTPS server but we don't know if they succeed in stealing valuable information. Some confidential data pass through this SSL channels so we hope it's not broken :~ 
+> 
+> The flag format is INS={some random string here}. 
+> 
+> pcapbleeding
+
```
Comment (dhanvi) [inline insomnihack-ctf-2016/crypto/pcapbleeding/README.md]: looks like @bcyrill didn't link the files in the readme

**Comment by dhanvi** [comment]
@bcyrill Thank You very much for the time spend by you to create the place holders 

If you wana join us come here https://gitter.im/ctfs/write-ups

File: insomnihack-ctf-2016/.gitignore (hunk 1 above)
Comment (bcyrill) [inline insomnihack-ctf-2016/.gitignore]: git lfs or any other preferences?

**Pull request merged**